engine/src/main/java/org/hibernate/validator/internal/util/logging/Log.java

```diff
@@ -640,4 +640,7 @@ UnexpectedTypeException getNoValidatorFoundForTypeException(String constraintTyp
 	@LogMessage(level = WARN)
 	@Message(id = 192, value = "Couldn't determine Java version from value %1s; Not enabling features requiring Java 8")
 	void unknownJvmVersion(String vmVersionStr);
+
+	@Message(id = 193, value = "%s is configured more than once via the programmatic constraint definition API.")
```

Comment:
Currently that message is only used when it's done multiple time on the **same** `ConstraintMapping` object. When done on multiple `ConstraintMapping` objects, that's the code I added on `ConstraintHelper` that takes over. I'm not sure that the programmatic API handles this kind of misuse for constraint mappings; does it? Should it?